Cab share/Login/LoginView.swift

```diff
@@ -6,12 +6,27 @@
 //
 
 import SwiftUI
+import FirebaseCore
+import FirebaseAuth
 
 struct LoginView: View {
     @State private var email = ""
     @State private var password = ""
-    @State private var signIn = false
+    @State private var signInView = false
+    @State private var userIsLoggedIn = false
+    @State private var userSignedUp = false
+    @State private var userSignedIn = false
```

Comment:
Make clear which var we use and delete unused ones.

Also if we are using MVVM Architecture it's better to put it in a ViewModel.